### PR TITLE
update to 0.7.0

### LINKED ISSUE #232: Prepare Release 0.70
We will make a new release before merging BGPsec support in to the rtrlib.

### PR DESCRIPTION
**Pull request by smlng**
### Contribution description

This PR prepares the next release of version 0.7.0, which should be the last before introducing initial BGPsec support, see #217.

### Issues/PRs references

fixes #232

### CODE REVIEW

**Comment by mroethke** [comment]
I would like to include #219 in this release. The implementation is almost done and I think I can finish it today.

**Comment by smlng** [comment]
I guess we have accumulated (and merged) enough changes to justify a new release, which this PR prepares. Please check change logs.

File: CHANGELOG [hunk 1]
```diff
@@ -3,6 +3,19 @@ https://github.com/rtrlib/rtrlib/commits/master
 
 Release History:
 
+* Version 0.7.0
+    - Ensure shadow tables are only freed when initialized
```
Comment (waehlisch) [inline CHANGELOG]: sounds like a fix. change wording.

File: CHANGELOG [hunk 2]
```diff
@@ -3,6 +3,19 @@ https://github.com/rtrlib/rtrlib/commits/master
...
+    - Fix SSH transport and host key verification
```
Comment (waehlisch) [inline CHANGELOG]: briefly explain to achieve what.

File: CHANGELOG [hunk 3]
```diff
@@ -3,6 +3,19 @@ https://github.com/rtrlib/rtrlib/commits/master
...
+    - Update tommyds implementation
```
Comment (waehlisch) [inline CHANGELOG]: to which version?

File: CHANGELOG [hunk 4]
```diff
@@ -3,6 +3,19 @@ https://github.com/rtrlib/rtrlib/commits/master
...
+    - Fix check ensuring uniqueness of group preferences
```
Comment (waehlisch) [inline CHANGELOG]: Fix bug in cache groups to ensure uniqueness of cache server preferences

File: CHANGELOG [hunk 5]
```diff
@@ -3,6 +3,19 @@ https://github.com/rtrlib/rtrlib/commits/master
...
+    - Fix race condition in rtr_stop
+    - Remove error PDU null termination check
+    - tools/rpki-rov: improve input validation
```
Comment (waehlisch) [inline CHANGELOG]: unclear

File: CHANGELOG [hunk 6]
```diff
@@ -3,6 +3,19 @@ https://github.com/rtrlib/rtrlib/commits/master
...
+    - Output log to stderr instead of stdout
```
Comment (waehlisch) [inline CHANGELOG]: Change output log: Log to stderr instead of stdout

File: CHANGELOG [hunk 7]
```diff
@@ -3,6 +3,19 @@ https://github.com/rtrlib/rtrlib/commits/master
...
+    - tools/rtrclient: several updates and refinements
```
Comment (waehlisch) [inline CHANGELOG]: unclear

File: CHANGELOG [hunk 8]
```diff
@@ -3,6 +3,19 @@ https://github.com/rtrlib/rtrlib/commits/master
...
+    - style: increase max line length to 120 characters
```
Comment (waehlisch) [inline CHANGELOG]: put at the end

File: CHANGELOG [hunk 9]
```diff
@@ -3,6 +3,19 @@ https://github.com/rtrlib/rtrlib/commits/master
...
+    - tools/rtrclient: add roa export functionality
```
Comment (waehlisch) [inline CHANGELOG]: be more concrete what is supported now

change order of items and put tools/ together.

Comment (smlng) [reply CHANGELOG]: okay will adapt,

**Comment by smlng** [comment]
@waehlisch addressed your comments, please check, thx.

**Comment by smlng** [comment]
rebased and squash, will merge after CI checks are green

**Pull request merged**